### ISSUE TITLE
Division by 0 error

### ISSUE THREAD

**Issue by Chinese1904** [pionl/laravel-chunk-upload#15]
I have noticed in my error logs that every now and then this error pops up. Just now I ran into it for the first time myself. 
What's weird is I just now tested a file which works on localhost, but I get division by 0 error in production.

After digging a little deeper the issue seems to be that if it is not a chunked upload, `$save->isFinished()` false is, then `$handler->getPercentageDone` throws this error. It might be due to a a slow upload connection why it works on local and not on production?

(I am not too sure if this issue comes up when it is a chunked upload)

```
Whoops, looks like something went wrong.
1/1
ErrorException in ContentRangeUploadHandler.php line 170:
Division by zero
in ContentRangeUploadHandler.php line 170
at HandleExceptions->handleError(2, 'Division by zero', '/home/clooudtv/httpdocs/vendor/pion/laravel-chunk-upload/src/Handler/ContentRangeUploadHandler.php', 170, array()) in ContentRangeUploadHandler.php line 170
at ContentRangeUploadHandler->getPercentageDone() in MediaController.php line 146
at MediaController->directUploadHandler(object(UploadMediaRequest)) in MediaController.php line 65
at MediaController->uploadHandler(object(UploadMediaRequest))
at call_user_func_array(array(object(MediaController), 'uploadHandler'), array(object(UploadMediaRequest))) in Controller.php line 55
at Controller->callAction('uploadHandler', array(object(UploadMediaRequest))) in ControllerDispatcher.php line 44
at ControllerDispatcher->dispatch(object(Route), object(MediaController), 'uploadHandler') in Route.php line 203
at Route->runController() in Route.php line 160
at Route->run() in Router.php line 559
at Router->Illuminate\Routing\{closure}(object(Request)) in Pipeline.php line 30
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in CheckReferral.php line 27
at CheckReferral->handle(object(Request), object(Closure)) in Pipeline.php line 148
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in NotBlocked.php line 24
at NotBlocked->handle(object(Request), object(Closure)) in Pipeline.php line 148
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in SubstituteBindings.php line 41
at SubstituteBindings->handle(object(Request), object(Closure)) in Pipeline.php line 148
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in VerifyCsrfToken.php line 65
at VerifyCsrfToken->handle(object(Request), object(Closure)) in Pipeline.php line 148
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in ShareErrorsFromSession.php line 49
at ShareErrorsFromSession->handle(object(Request), object(Closure)) in Pipeline.php line 148
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in StartSession.php line 64
at StartSession->handle(object(Request), object(Closure)) in Pipeline.php line 148
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in AddQueuedCookiesToResponse.php line 37
at AddQueuedCookiesToResponse->handle(object(Request), object(Closure)) in Pipeline.php line 148
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in EncryptCookies.php line 59
at EncryptCookies->handle(object(Request), object(Closure)) in Pipeline.php line 148
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in Pipeline.php line 102
at Pipeline->then(object(Closure)) in Router.php line 561
at Router->runRouteWithinStack(object(Route), object(Request)) in Router.php line 520
at Router->dispatchToRoute(object(Request)) in Router.php line 498
at Router->dispatch(object(Request)) in Kernel.php line 174
at Kernel->Illuminate\Foundation\Http\{closure}(object(Request)) in Pipeline.php line 30
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in TransformsRequest.php line 30
at TransformsRequest->handle(object(Request), object(Closure)) in Pipeline.php line 148
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in CheckForMaintenanceMode.php line 46
at CheckForMaintenanceMode->handle(object(Request), object(Closure)) in Pipeline.php line 148
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in Pipeline.php line 102
at Pipeline->then(object(Closure)) in Kernel.php line 149
at Kernel->sendRequestThroughRouter(object(Request)) in Kernel.php line 116
at Kernel->handle(object(Request)) in index.php line 57
```

**Comment by pionl** [comment]
Hi, the `$handler->getPercentageDone` should not be used when upload is not completed (means `isFinished` is false. However, I will check the values before computing percentage.

Thank you for the report :)

**Comment by pionl** [comment]
Thank you for the report, fixed in `1.0.4`
